pkgs/applications/version-management/git-and-tools/git/default.nix

```
@@ -311,6 +311,9 @@ stdenv.mkDerivation {
     # Tested to fail: 2.18.0
     disable_test t9902-completion "sourcing the completion script clears cached --options"
 
+    # flaky test which does fails when manually building
```

```suggestion
    # Flaky tests:
```

In case we add future tests and "manually building" doesn't make sense for me (there's kinda only one way to build stuff).

---

I also wouldn't have known how to disable single tests.